src/System.ServiceProcess.ServiceController/tests/System.ServiceProcess.ServiceController.Tests/System.ServiceProcess.ServiceController.Tests.csproj

```diff
@@ -29,9 +29,9 @@
     </ProjectReference>
   </ItemGroup>
   <ItemGroup Condition="'$(TargetsWindows)'=='true'">
-    <ProjectReference Include="..\NativeTestService\NativeTestService.vcxproj"> 
+    <ProjectReference Include="..\NativeTestService\System.ServiceProcess.ServiceController.TestNativeService.vcxproj"> 
```

Comment:
This test project is also calling create process with this referenced exe so the tests will break with this rename.